frontend/src/components/molecules/NotesTab/NotesTab.js

```diff
@@ -0,0 +1,68 @@
+import "./NotesTab.css";
+import Heading from "../../atoms/Heading/Heading";
+import { useState } from "react";
+import Note from "../../atoms/Note/Note";
+import TextField from "../TextField/TextField";
+import Button from "../../atoms/Button/Button";
+import { INITIAL_NOTE_DATA } from "../../../utils/constants";
+
+function NotesTab() {
+  const [notes, setNotes] = useState([]);
+  const [noteContent, setNoteContent] = useState("");
+
+  const onContentChange = (newContent) => {
+    setNoteContent(newContent);
+  };
+
+  const onSendNotes = () => {
+    if (noteContent.length === 0) {
+      return;
+    }
+
+    let newNote = {
+      ...INITIAL_NOTE_DATA,
+      content: noteContent,
+      date: Date.now(),
```

Comment:
Be aware that this is only a shallow copy of `INITIAL_NOTE_DATA`. That means that when you add a speaker to `newNote`, it is also added to `INITIAL_NOTE_DATA` and all other shallow copies of it, since all refer to the same array.

The same problem might apply to other `INITIAL_ ... _DATA` in `constants.js`.

It might be ok for the time being, since I can imagine that the data handling could change later on. But keep this in mind.

Here is a simplified example for the problem:

```
const a = {"arr": []}
const b = {...a}

console.log(a.arr) // logs: []
console.log(b.arr) // logs: []

b.arr.push(1)

console.log(a.arr) // logs: [1]
console.log(b.arr) // logs: [1]
```

Possible solutions for the future could be to either make a deep copy every time you need a new object, or simply create a new object (preferably).